codec-http/src/test/java/io/netty/handler/codec/http/websocketx/WebSocketServerHandshaker13Test.java

```diff
@@ -135,7 +129,7 @@ private static void testUpgrade0(EmbeddedChannel ch, WebSocketServerHandshaker13
 
             handshaker.handshake(ch, req);
 
-            ByteBuf resBuf = ch.readOutbound();
+            Buffer resBuf = ch.readOutbound();
```

Comment:
The `ReferenceCountUtil.release(res);` call doesn't work on these. Use try-with-resources.